PaaConditionalMenu/src/paaconditionalmenu/PaaConditionalMenu.java

```diff
@@ -0,0 +1,86 @@
+package paaconditionalmenu;
+
+import java.util.Scanner;   // Needed for Scanner class
+import java.util.Date;      // Needed for Date and Time
+import java.text.DateFormat;    // Needed for Date and Time
+import java.text.SimpleDateFormat;  // Needed for Date and Time
+import java.util.Calendar;      // Needed for Date and Time
+
```

Comment:
Overall, you did a nice job documenting.  I was not familiar with how to show the current date and time.  I could easily follow along.  I noticed in the book they suggest adding a documentation comment just before the class header.  That might have been helpful.